src/Compilers/CSharp/Portable/Lowering/SynthesizedMethodBaseSymbol.cs

```diff
@@ -46,11 +46,15 @@ internal abstract class SynthesizedMethodBaseSymbol : SourceMemberMethodSymbol
 
             this.MakeFlags(
                 methodKind: MethodKind.Ordinary,
+                refKind: baseMethod.RefKind,
                 declarationModifiers: declarationModifiers,
                 returnsVoid: baseMethod.ReturnsVoid,
+                hasAnyBody: false,
```

Comment:
>hasAnyBody: false, [](http://example.com/codeflow?start=16&length=18)

It feels like `true` is more appropriate for this symbol.

Reply:
Would it make sense to defer to the original baseMethod? (for this and isExpressionBodied)?

Reply:
> Would it make sense to defer to the original baseMethod? (for this and isExpressionBodied)?

I don't think so. Definitely not for ```isExpressionBodied```. And ``baseMethod``` could be `extern` (i.e. without body). So, it really doesn't matter what the ```baseMethod``` has.

Reply:
would it be acceptable to rename this to hasBlockOrExpressionBody (thus clearly spelling out what it means, and then making it clearer about what values should be used in these synthesized locations)?  If we add 'true' for some of these cases, i actually genuinely am not sure how to doc what the HasAnyBody concept gets you.

Reply:
Will move this to potentially another PR.  Can keep hasAnyBody for now here.

Reply:
> would it be acceptable to rename this to hasBlockOrExpressionBody

I think that would be inappropriate. I like the fact that the name doesn't imply any syntax involvement, and would prefer to keep it this way.